Payment/index.ts

```diff
@@ -1,4 +1,4 @@
-import { Card as PaymentCard } from "./Card"
+import { Card as PaymentCard } from "../Card"
```

Comment:
Card is moved to the top. There is no longer need to re export it on payments namespace.